### ISSUE TITLE
Accessing markdown variables inside template

### ISSUE THREAD

**Issue by SonniesEdge** [trulia/hologram#258]
It would be great if we could access the documentation variables inside a template. So, in addition to the `<%= rendered_example %>` and `<%= code_example %>` variables, we might have `<%= category %>` and `<%= name %>`.

As a use case, being able to assign the current name as an ID would really help with our automated visual regression testing workflow.

**Comment by ffoodd** [comment]
I agree, I'd need an ID to add an accessible hide/show system on templates — which means using `[aria-controls="ID"]`.

And there's probably other use cases out there :)
